resources/fieldsets/call_to_action.yaml

```diff
@@ -0,0 +1,112 @@
+title: 'Call to action'
+fields:
+  -
+    import: component_settings
+  -
+    handle: title_heading
+    field:
+      input_type: text
+      antlers: false
+      type: text
+      display: 'Title - Heading'
+      icon: text
+      listable: hidden
+      instructions_position: above
+      visibility: visible
+      replicator_preview: true
+      hide_display: false
+      width: 50
+  -
+    handle: title_subheading
```

Comment:
I don't see the relevance of the `title_` prefix on heading and subheading but that's just an opinion

Reply:
On the other hand, i'd rather see title & subtitle. The _heading is usually used for choosing actual heading tags on titles etc.